docs/advanced/repo_subdirectories.rst

```diff
@@ -0,0 +1,28 @@
+.. _`subdirectories`:
+
+Organizing cookiecutter templates in subdirectories
+===================================================
+
+You can also organize one or more templates in a single repo, like this::
+
+    https://github.com/user/repo-name.git
+        └── subdirectory-name/
+            ├── {{cookiecutter.project_slug}}/
+            └── cookiecutter.json
+
+And then, using the cli::
+
+    cookiecutter https://github.com/user/repo-name.git --subdirectory="subdirectory-name"
+
+Cookiecutter follows this high-level algorithm
+
+1) Copy/clone/unzip a directory into (by default) ``~/.cookiecutter/``
```

Comment:
Formatting of this block looks very strange after Sphinx rendering.
You can render docs locally to look how it will be on docs site.
Here is screenshot.

![Screenshot from 2019-09-04 12-15-39](https://user-images.githubusercontent.com/1916759/64242207-cd748200-cf0d-11e9-8f31-2c020723ba6f.png)

P.S. Writting docs is hard :)

Reply:
Thanks, fixed!